settlement/dymension/dymension.go

```diff
@@ -221,55 +227,59 @@ WaitForBatchAcceptance:
 				utils.SubmitEventOrPanic(d.ctx, d.pubsub, heatlhEventData,
 					map[string][]string{settlement.EventTypeKey: {settlement.EventSettlementHealthStatus}})
 				// Sleep to allow context cancellation to take effect before retrying
+
+				//TODO(mtsitrin): I think it should panic here, if it fails after retries. same as in DA submission
```

Comment:
what would be the benefit of that? 

As I see it we're in a constant retry. we have nothing else to do but retry and emit health/unhealthy events according to the situation.

Reply:
I think we need to define whether we want an endless retries loop, or panic after some threshold
currently it's a mix. it will panic if DA fails for too many retries, but won't for SL.

For sure we can leave it as is, but it's making all the retries management useless (as it's in an endless loop)
The RPC isn't responsive in case of unhealthy, so not sure what the difference is.